iree/compiler/Codegen/Common/test/convert_to_destination_passing_style.mlir

```diff
@@ -575,3 +575,31 @@ func.func @fill_matmul_exp() {
 //       CHECK:   %[[MATMUL:.+]] = linalg.matmul
 //       CHECK:   linalg.generic
 //  CHECK-SAME:       outs(%[[MATMUL]]
+
+// -----
+
+func @cumsum__2x2x2x2x2x2x2() {
```

Comment:
Not sure I see the connection between the code changes and the lit test. None of the cases have the `arith.constant` as the out operand?

Reply:
good catch.. fixed!

Reply:
Maybe use a value other than 0.0 for the constant in the lit test. Otherwise might give a wrong impression that its ok to fill by 0 always.